easybuild/easyconfigs/u/UCX/UCX-1.16.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'ConfigureMake'
+
+name = 'UCX'
+version = '1.16.0'
+
+homepage = 'https://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openucx/ucx/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+patches = [
+    'UCX-1.13.1-dynamic_modules.patch',
+]
+checksums = [
+    {'ucx-1.16.0.tar.gz': 'f73770d3b583c91aba5fb07557e655ead0786e057018bfe42f0ebe8716e9d28c'},
+    {'UCX-1.13.1-dynamic_modules.patch': '00874687bd90b795fff61aaa183f6c6bea2210aa1003b28f23d9ebf7066f8782'},
+]
+
+# Uncomment when updating to final release
+# sources = ['%(namelower)s-%(version)s.tar.gz']
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Autotools', '20231222'),
+    ('pkgconf', '2.2.0'),
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+dependencies = [
+    ('zlib', '1.3.1'),
+    ('numactl', '2.0.18'),
+]
+
+configure_cmd = "contrib/configure-release"
```

Comment:
@SebastianAchilles Please change this to use the standard configure script, see #20428